core/ControlMacros.carp

```diff
@@ -133,6 +133,13 @@ Example:
 (defmacro forever-do [:rest forms]
   (list 'while true (cons 'do forms)))
 
+ (doc ignore-do 
+ ("Wraps side-effecting `forms` in a `do`, " false)
+ "ignoring all of their results."
+ "In other words, executes `forms` only for their side effects.")
+ (defmacro ignore-do [:rest forms]
+   (cons 'do (expand (apply ignore-all forms))))
```

Comment:
Shouldn't this use `ignore*` instead? I can't find `ignore-all`. (Maybe add a test case to catch this also.)

Reply:
Good catch--forgot to update after a rename.

Reply:
Fixed! + Added a test.